guidelines/releases-guidelines/02-release-process.md

```diff
@@ -37,13 +37,13 @@ Define release jobs on the `master` branch in the `test-infra` repository. To en
 		        value: release-0.9
 		```
 
-1. Ensure that tests for the release jobs exist. Release tests usually iterate through all release versions and run tests for them.
+4. Ensure that tests for the release jobs exist. Release tests usually iterate through all release versions and run tests for them.
```

Comment:
But why not use automatic numbering? Markdown viewers will render the correct numbers automatically.

Reply:
Automatic numbering seems to work just fine